apps/dcos_l4lb/src/dcos_l4lb_mgr.erl

```diff
@@ -460,9 +478,25 @@ healthy_vips(VIPs, Nodes, Tree)
     VIPs;
 healthy_vips(VIPs, Nodes, Tree) ->
     Agents = agents(VIPs, Nodes, Tree),
-    lists:map(fun ({VIP, BEs}) ->
+
+    HealthyVIPs = lists:map(fun ({VIP, BEs}) ->
         {VIP, healthy_backends(BEs, Agents)}
-    end, VIPs).
+    end, VIPs),
+
+    AllBackends =
+        lists:sum([length(BEs) || {_VIP, BEs} <- VIPs]),
+    prometheus_gauge:set(l4lb, backends, [], AllBackends),
+
+    HealthyBackends =
+        lists:sum([length(BEs) || {_VIP, BEs} <- HealthyVIPs]),
+    prometheus_gauge:set(
+        l4lb, unreachable_backends, [],
+        AllBackends - HealthyBackends),
+
+    UnrecheableVIPs = [VIP || {VIP, BEs} <- HealthyVIPs, length(BEs) =:= 0],
```

Comment:
How did you test it? `healthy_backends/2` returns all backends if all of them are not reachable.